### PR TITLE
docs: add plugin model docs

### PR DESCRIPTION
**Pull request by mappjzc**
Add plugin model docs.

Nddtfjiang <zhicheng.jiang@merico.dev>

### CODE REVIEW

**Review by long2ice — DISMISSED** [review]

File: docs/DataModels/DevLakeDomainLayerSchema.md [hunk 1]
```diff
@@ -46,6 +46,21 @@ Tables that end with WIP are still under development.
 
 <br/>
 
+## Get all domain layer model info.
+
+All domain layer models can be accessed by the following methods
```
Comment (klesh) [inline docs/DataModels/DevLakeDomainLayerSchema.md]: method, no `s`

File: docs/DataModels/DevLakeDomainLayerSchema.md [hunk 2]
```diff
@@ -46,6 +46,21 @@ Tables that end with WIP are still under development.
...
+```golang
+import "github.com/apache/incubator-devlake/models/domainlayer/domaininfo"
+
+domaininfo := domaininfo.GetDomainTablesInfo()
+for _, table := range domaininfo {
+  // do something 
+}
+```
+
+If you want to learn infomation about plugin model,please visit [PluginImplementation](https://devlake.apache.org/docs/DeveloperManuals/PluginImplementation)
```
Comment (klesh) [inline docs/DataModels/DevLakeDomainLayerSchema.md]: learn **more** about plugin **models**

File: docs/DeveloperManuals/PluginImplementation.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ A plugin mainly consists of a collection of subtasks that can be executed by Dev
 3. [PluginTask](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_task.go) enables a plugin to prepare data prior to subtask execution
 4. [PluginApi](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_api.go) lets a plugin exposes some self-defined APIs
 5. [Migratable](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_db_migration.go) is where a plugin manages its database migrations 
+6. [PluginModel](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_model.go) allows other plugins to get the model information of all database tables of the current plugin through the GetTablesInfo() method.If you want to know more infomation about model of domain layer,please visit [DomainLayerSchema](https://devlake.apache.org/docs/DataModels/DevLakeDomainLayerSchema/)
```
Comment (klesh) [inline docs/DeveloperManuals/PluginImplementation.md]: ... If you need to access Domain Layer Models, ..

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**